src/Containers.jsx

```diff
@@ -372,42 +372,42 @@ class Containers extends React.Component {
         window.removeEventListener('resize', this.onWindowResize);
     }
 
-    renderRow(containersStats, container, containerDetail, localImages) {
+    renderRow(containersStats, container, localImages) {
         const containerStats = containersStats[container.Id + container.isSystem.toString()];
-        const image = container.Image;
-        let healthcheck = "";
+        const image = container.ImageName;
         let localized_health = null;
 
+        // this needs to get along with stub containers from image run dialog, where most properties don't exist yet
         // HACK: Podman renamed `Healthcheck` to `Health` randomly
         // https://github.com/containers/podman/commit/119973375
-        if (containerDetail)
-            healthcheck = containerDetail.State.Health ? containerDetail.State.Health.Status : containerDetail.State.Healthcheck.Status;
+        const healthcheck = container.State?.Health?.Status ?? container.State?.Healthcheck?.Status;
+        const status = container.State?.Status ?? "";
```

Comment:
These 2 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1350-20230712-152457-eee84e1f-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)